.github/ISSUE_TEMPLATE/engineering-onboarding.md

```diff
@@ -42,13 +42,16 @@ _Goal: I can ssh into gravity (staging and production). I can PR to a repo in Ar
 
 #### Reading/Learning:
 
-- [ ] Take a tour of Artsy. Log into [staging.artsy.net](https://staging.artsy.net) with your admin account and click around without fear. Visit each of the top-nav pages. Bid in an auction. Make an inquiry. _Ask your mentor or others for help with any of these concepts. Have them show you CMS and some internal apps, like Ohm or Torque._
-- [ ] Learn how code goes from your local machine --> production. _Talk to your mentor. Have them explain gravity's deployment process. Read about our [staging environment](https://github.com/artsy/gravity/blob/master/doc/StagingEnvironment.md).
-- [ ] Reading: [Working with the API](https://github.com/artsy/potential/blob/master/platform/WorkingWithGravity.md)
+- [ ] Take a tour of Artsy. Log into [staging.artsy.net](https://staging.artsy.net) with your admin account (i.e. your `@artsymail.com` account) and click around without fear. Visit each of the top-nav pages. Bid in an auction. Make an inquiry. _Ask your mentor or others for help with any of these concepts. Have them show you CMS and some internal apps, like Ohm or Torque._
+- [ ] Besides [Artsy’s base values](https://github.com/artsy/README/blob/master/culture/what-is-artsy.md#artsy-values), the engineering team has its own additional [set of principles](https://github.com/artsy/README/blob/master/culture/engineering-principles.md)
+- [ ] Learn how code goes from your local machine --> production. _Talk to your mentor. Have them explain the Gravity, Metaphysics, and/or Force/Reaction deployment process. Read about our [staging environment](https://github.com/artsy/gravity/blob/master/doc/StagingEnvironment.md).
 - [ ] Scan our list of preferred technologies. Dig in where necessary.
   - [ ] [Front-end technologies](https://github.com/artsy/README/blob/master/practices/front-end.md)
   - [ ] [Back-end/Platform technologies](https://github.com/artsy/README/blob/master/practices/platform.md)
   - [ ] iOS/React Native? #TODO
+- [ ] Reading: [Best of Our Blog](https://github.com/artsy/README/blob/master/recommendations/blog.md)
+- [ ] Reading: [Internal and external tech learning resources](https://github.com/artsy/README/blob/master/recommendations/tech-learning.md)
+- [ ] Reading: [Working with the API](https://github.com/artsy/potential/blob/master/platform/WorkingWithGravity.md)
```

Comment:
I feel like this is maybe too specific to have everybody read during onboarding, additionally it’s already linked to from the above [Back-end/Platform technologies](https://github.com/artsy/README/blob/master/practices/platform.md) link. What do you think?

Reply:
Sure! Let's remove.